gamemodes/terrortown/entities/weapons/weapon_ttt_beacon.lua

```diff
@@ -0,0 +1,245 @@
+---
+-- @class SWEP
+-- @section weapon_ttt_beacon
+
+if SERVER then
+	AddCSLuaFile()
+end
+
+DEFINE_BASECLASS "weapon_tttbase"
```

Comment:
```suggestion
DEFINE_BASECLASS("weapon_tttbase")
```

Reply:
I think this can be removed as the other entities also do not use this anymore.

But dont know if this actually changes anything. Would have to look that up

Reply:
Right now all our entities and weapons do this:

https://github.com/TTT-2/TTT2/blob/master/gamemodes/terrortown/entities/weapons/weapon_ttt_wtester.lua#L12

Reply:
my main concern is twofold:
	- it's not used how the wiki tells us to e.g. `DEFINE_BASECLASS( "my_weapon" )` [1]
	- it breaks github's syntax highlighting (it doesn't break treesitter nor vim's default highlighting; so maybe this is just a github problem. Still not ideal)

if we can eliminate this usage completely that would be fine by me as well.

[1] https://wiki.facepunch.com/gmod/Global.DEFINE_BASECLASS

Reply:
This should be safe to remove, since we are not using it in here

Reply:
ok, but we still have this line in all (or most) other entities. Maybe someone else can remove them then